Makefile

```diff
@@ -10,8 +10,8 @@ include ${ENV_FILE}
 export $(shell sed 's/=.*//' ${ENV_FILE})
 endif
 
-CONTAINER_BUILDER=podman
-CONTAINER_DOCKERFILE=Dockerfile
+CONTAINER_BUILDER?=podman
```

Comment:
Thanks, not sure how I missed the `-e`  flag. TIL

https://media.giphy.com/media/w89ak63KNl0nJl80ig/giphy.gif

I will improve the docs as part of this PR.

Reply:
Perfect! Thank you very much!

Reply:
Hi @bartoszmajsak did you have the time to review this? If you want we can take a look at this, it would be nice to merge this as soon as possible cause it was a really nice improvement!
Thanks in advance!

Reply:
@lucferbux I was away on vacation, I will finalize it this week.

Reply:
Great, thank you very much!

Reply:
@bartoszmajsak we are doing a big revamp of our docs right now, feel free to [take a look](https://github.com/opendatahub-io/odh-dashboard/pull/1851), rebase once it's merged and update accordingly, thanks a lot!